File: Oregon 9-1-1/Oregon 9-1-1.Android.NativeActivity/callInfo.cpp
#include "Oregon911.h"

callInfo::callInfo(unsigned int _id, std::string _callSum, unsigned int _callNumber, std::string _address, std::string _agency, std::string _station, std::string _units, unsigned int _unitCount, bool _active, double _lat, double _lon, timestamp _ts)
{
	id = _id;
	callSum = _callSum;
	callNumber = _callNumber;
	address = _address;
	agency = _agency;
	station = _station;
	units = _units;
	unitCount = _unitCount;
	active = _active;
	lat = _lat;
	lon = _lon;
	ts = _ts;
}

callInfo::callInfo()
{
	id = 0;
	callNumber = 0;
	unitCount = 0;
	active = false;
	lat = 0.0;
	lon = 0.0;
}

callInfo::~callInfo()
{

}

unsigned int callInfo::getid()
{
	return id;
}

std::string callInfo::getCallSum()
{
	return callSum;
}

unsigned int callInfo::getCallNumber()
{
	return callNumber;
}

std::string callInfo::getAddress()
{
	return address;
}

std::string callInfo::getAgency()
{
	return agency;
}

std::string callInfo::getStation()
{
	return station;
}

std::string callInfo::getUnits()
{
	return units;
}

unsigned int callInfo::getUnitCount()
{
	return unitCount;
}

bool callInfo::getActive()
{
	return active;
}

double callInfo::getLat()
{
	return lat;
}

double callInfo::getLon()
{
	return lon;
}

timestamp callInfo::getTimeStamp()
{
	return ts;
}

void callInfo::setid(const unsigned int& _id)
{
	id = _id;
}

void callInfo::setCallSum(const std::string& _callsum)
{
	callSum = _callsum;
}

void callInfo::setCallNumber(const unsigned int& _callnumber)
{
	callNumber = _callnumber;
}

void callInfo::setAddress(const std::string& _address)
{
	address = _address;
}

void callInfo::setAgency(const std::string& _agency)
{
	agency = _agency;
}

void callInfo::setStation(const std::string& _station)
{
	station = _station;
}

void callInfo::setUnits(const std::string& _units)
{
	units = _units;
}

void callInfo::setUnitCount(const unsigned int& _unitcount)
{
	unitCount = _unitcount;
}

void callInfo::setActive(const bool& _active)
{
	active = _active;
}

void callInfo::setLat(const double& _lat)
{
	lat = _lat;
}

void callInfo::setLon(const double& _lon)
{
	lon = _lon;
}

void callInfo::setTimeStamp(const timestamp& _ts)
{
	ts = _ts;
}

File: Oregon9-1-1/app/src/main/java/cad/oregon911/net/callinfo.java
package cad.oregon911.net;

/**
 * Created by Brandan on 12/28/2015.
 */
public class callinfo {

    public callinfo() {
        this.id = 0;
        this.callNumber = 0;
        this.active = false;
        this.lat = 0.0;
        this.lon = 0.0;
        ts = new timestamp();
    }

    public callinfo(int id, String callSum, int callNumber, String address, String agency, String station, String units, int unitCount, boolean active, double lat, double lon, timestamp ts) {
        this.id = id;
        this.callSum = callSum;
        this.callNumber = callNumber;
        this.address = address;
        this.agency = agency;
        this.station = station;
        this.units = units;
        this.unitCount = unitCount;
        this.active = active;
        this.lat = lat;
        this.lon = lon;
        this.ts = ts;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getCallSum() {
        return callSum;
    }

    public void setCallSum(String callSum) {
        this.callSum = callSum;
    }

    public int getCallNumber() {
        return callNumber;
    }

    public void setCallNumber(int callNumber) {
        this.callNumber = callNumber;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getAgency() {
        return agency;
    }

    public void setAgency(String agency) {
        this.agency = agency;
    }

    public String getStation() {
        return station;
    }

    public void setStation(String station) {
        this.station = station;
    }

    public String getUnits() {
        return units;
    }

    public void setUnits(String units) {
        this.units = units;
    }

    public int getUnitCount() {
        return unitCount;
    }

    public void setUnitCount(int unitCount) {
        this.unitCount = unitCount;
    }

    public boolean isActive() {
        return active;
    }

    public void setActive(boolean active) {
        this.active = active;
    }

    public double getLat() {
        return lat;
    }

    public void setLat(double lat) {
        this.lat = lat;
    }

    public double getLon() {
        return lon;
    }

    public void setLon(double lon) {
        this.lon = lon;
    }

    public timestamp getTs() {
        return ts;
    }

    public void setTs(timestamp ts) {
        this.ts = ts;
    }

    private int id;
    private String callSum;
    private int callNumber;
    private String address;
    private String agency;
    private String station;
    private String units;
    private int unitCount;
    private boolean active;
    private double lat;
    private double lon;
    private timestamp ts;
}
